docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,50 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- Added new
+  [`experimentalSkipDomainInjection`](/guides/references/experiments#Experimental-Skip-Domain-Injection)
+  flag to disable Cypress from setting `document.domain` on injection. Addresses
+  [#2367](https://github.com/cypress-io/cypress/issues/2367),
+  [#23958](https://github.com/cypress-io/cypress/issues/23958),
+  [#24290](https://github.com/cypress-io/cypress/issues/24290), and
+  [#24418](https://github.com/cypress-io/cypress/issues/24418).
+- Added new
+  [`experimentalMemoryManagement`](/guides/references/experiments#Configuration)
+  flag to improve memory management in Chromium-based browsers. Addresses
+  [#23391](https://github.com/cypress-io/cypress/issues/23391).
+- [`.as`](/api/commands/as) now accepts an options argument, allowing an alias
+  to be stored as a "query" or "static" value. Addresses
+  [#25173](https://github.com/cypress-io/cypress/issues/25173).
+- Added official support for Vite 4 projects for component testing. Addresses
+  [#24969](https://github.com/cypress-io/cypress/issues/24969).
+- The `cy.log()` command will now display a line break where the `\n` character
+  is used. Addresses
+  [#24964](https://github.com/cypress-io/cypress/issues/24964).
+- `.jsx`/`.tsx` file extensions will be utilized when generating a new empty
+  spec file inside component testing if the project contains at least one file
+  with those extensions. Addresses
+  [#24495](https://github.com/cypress-io/cypress/issues/24495).
+- Support `data-qa` selector in Selector Playground in addition to `data-cy`,
+  `data-test` and `data-testid`. Addresses
+  [#25305](https://github.com/cypress-io/cypress/issues/25305).
+
+**Bugfixes:**
+
+- Fixed an issue where component tests could incorrectly treat new major
+  versions of certain dependencies as supported. Fixes
+  [#25379](https://github.com/cypress-io/cypress/issues/25379).
+- Fixed an issue where spaces on new lines weren't kept in the test runner.
```

Comment:
```suggestion
- Fixed an issue where spaces on new lines in the Command Log were not maintained.
```

Reply:
Honestly this entry & [#24964](https://github.com/cypress-io/cypress/issues/24964). sound almost identical. could we swizzle about honoring command long formatting and reference both?

Reply:
addressed in 2a4b1b2fa